src/node/quote.cpp

```diff
@@ -231,6 +255,62 @@ namespace ccf
     return QuoteVerificationResult::Verified;
   }
 
+  QuoteVerificationResult verify_tcb_version_against_store(
+    ccf::kv::ReadOnlyTx& tx, const QuoteInfo& quote_info)
+  {
+    if (quote_info.format != QuoteFormat::amd_sev_snp_v1)
+    {
+      return QuoteVerificationResult::Verified;
+    }
+
+    pal::PlatformAttestationMeasurement d = {};
+    pal::PlatformAttestationReportData r = {};
+    pal::verify_quote(quote_info, d, r);
```

Comment:
This is also strange, why do we combine verification and lookup from the store, they should be composable operations.

Reply:
This pattern has been true for a long-time, and I don't think we should tackle it here. I can understand why the initial API may have been written as "you can only retrieve verified values", but I think now that's a mistake - verification is expensive and potentially async, get is local and trivial. So these APIs should be revisited, but not in this PR.